mediacenter/views.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.admin.views.decorators import staff_member_required
+from django.core.urlresolvers import reverse_lazy
+from django.views.generic import (CreateView, DeleteView, DetailView, ListView,
+                                  UpdateView)
+
+from .models import Document
+
+
+class Index(ListView):
+    model = Document
+    template_name = 'mediacenter/index.html'
+    paginate_by = 10
+index = Index.as_view()
+
+
+class DocumentDetail(DetailView):
```

Comment:
If you don't plan to add anything to these classes, you might find shorter to pass the model directly to the `as_view()` method within the URLs. Does it beak the MVC pattern? Sure, but it's already broken anyway ;)